### ISSUE TITLE
Sudden reduction of loss

### ISSUE THREAD

**Issue by alizezhu** [MIC-DKFZ/nnUNet#244]
HI Fabian,
I have a strange reduction of loss during training, at around 900 epochs. I could not figure out why the reduction occurred. I tested the independent testing set, it worked fine.
Do you perhaps have an insight?

![image](https://user-images.githubusercontent.com/8353480/86225854-1a0b0700-bb8b-11ea-8ef1-8c59840df0d6.png)

I also wonder if the Dice is for validation set or the training set?
Thanks for the fabulous work!

**Comment by FabianIsensee** [comment]
Hi,
great to hear you enjoy our work :-)
I have seen this kind of behavior before but I also don't know what causes it. When you look at the evaluation metric (green) there seems to be nothing happening there, so whatever it is it does not affect the Dice score. Because of that I would also not worry too much about it. But just in case you figure out what this is I would be interested in an explanation ;-)
Best,
Fabian

**Comment by alizezhu** [comment]
Thanks Fabian for your answer! I assume that the green Dice curve is regarding the validation set (not the training set)? Do you think in this case the performance has saturated?

**Comment by FabianIsensee** [comment]
The red line is the validation loss and it its decreasing just like the training loss, so here for some reason a reduction in loss is not reflected in an improvement in the metric. Difficult to say why that is.

Anyways, looking at your graphs it seems like overfitting is not a problem. This is probably a large dataset. You could consider running a multi-GPU training to increase the batch size and maybe squeeze out some more performance. This will be more complicated though. I don't think this training will substantially benefit from more epochs.
